### PR TITLE
Create short-term production DSA server

### PR DESCRIPTION
**Pull request by BrunoGrandePhD**
The `dgutman-dsa-server-prod-ec2.yaml` stack being added here is a duplicate of `dgutman-htan-new2-ec2.yaml`. This new EC2 instance will be used as a short-term production server until we deploy something long-term (see IT-1214). The original stack will remain for now as a development server. 

For context, DSA stands for the Digital Slide Archive, which is an imaging file viewer being hosted for the HTAN project. 

----

PR Checklist:
- [x] Describe your resource request in your commit message
- [x] (Not applicable) Use the latest version of an example template 
- [x] Setup pre-commit and run the validators (info in README.md)

### CODE REVIEW

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (zaro0508) [inline config/prod/dgutman-dsa-server-prod-ec2.yaml:14]: this email is tied to parkmycloud and allow the OwnerEmail to manage this instance in PMC.  if it's not set to dgutman then he will not have access to start/stop this instance.

**Review by zaro0508 — CHANGES_REQUESTED** [review]
general Sage IT disclosure: it's NOT a good idea to use anything in sandbox for production.  But we also do't prevent people from shooting themselves in the foot either.

**Comment by BrunoGrandePhD** [comment]
@zaro0508: Thanks for the heads-up. I'm aware of the limitations of `sandbox`. Unfortunately, until I spin up another AWS account for this project and see progress on [IT-1214](https://sagebionetworks.jira.com/browse/IT-1214), this is the only way to get something up and running before the data release on May 17th. We were given little time to deploy a true production stack, so we will relay the risks of using `sandbox` to our collaborators.

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (zaro0508) [reply config/prod/dgutman-dsa-server-prod-ec2.yaml:14]: i'm not sure how that happened but it looks like dgutman has a sage jumpcloud account.

**Review by zaro0508 — APPROVED** [review]
OK, this is fine with me.  just waiting for review from @ahayden

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (ahayden) [reply config/prod/dgutman-dsa-server-prod-ec2.yaml:14]: We onboarding him as a contract worker

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (ahayden) [inline config/prod/dgutman-dsa-server-prod-ec2.yaml:3]: Please rename this to something like 'dgutman-dsa-server-prerprod-ec2`, as its not for prod and I don't want us to be confused about that later.

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (ahayden) [inline config/prod/dgutman-dsa-server-prod-ec2.yaml:7]: Does this web server need 80, or does this just for a redirect?

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (ahayden) [inline config/prod/dgutman-dsa-server-prod-ec2.yaml:20]: This is an old AMIId... should probably be v1.0.9 from here: https://ami.sageit.org/

**Review by ahayden — CHANGES_REQUESTED** [review]
A few changes and questions... Please confirm with Gov the classification of the data set

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (BrunoGrandePhD) [reply config/prod/dgutman-dsa-server-prod-ec2.yaml:20]: I wasn't aware of this AMI dashboard. Thanks for the pointer!

File: config/prod/dgutman-dsa-server-prod-ec2.yaml
Comment (BrunoGrandePhD) [reply config/prod/dgutman-dsa-server-prod-ec2.yaml:7]: Here's the response I got from our collaborator:

> For Redirect and for initial certificate grabbing with certbot

Comment (milen-sage) [reply config/prod/dgutman-dsa-server-prod-ec2.yaml]: @ahayden regarding data governance - this instance will only contain imaging data
- these data are not access-controlled (i.e. they are to be publicly released)
- all HTAN data that are not access-controlled have, by grant contract, been de-identified by data contributors (i.e. no PHI)
- PHI data can leak only accidentally (e.g. image header metadata left over by a data contributor); we are not legally liable if that happens. Even though at the HTAN DCC we are not required, we do at present review images (and accompanying metadata) for PHI prior to public data release.

In all these respects, this new instance will host the same data as the instance hosting the existing DSA deployment.

**Comment by BrunoGrandePhD** [comment]
I'm going to deploy this pre-production server in a separate AWS account. 

See https://github.com/ncihtan/aws-htan-infra/pull/2.

**Pull request closed without merging**
